11-functions.Rmd

```diff
@@ -5,6 +5,121 @@
 ottrpal::set_knitr_image_path()
 ```
 
-```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Major point!! example image"}
+```{r, fig.align='center', fig.alt = "Learning objectives are to be able to: Recognize when it is appropriate to make a custom function to aid in the reproducibility of an analysis. Create functions that are reproducible and useable by others. What aspects of a function make them useable by others. Well documented. Follow a style. Properly engineered", out.width="100%", echo = FALSE}
```

Comment:
```suggestion
```{r, fig.align='center', fig.alt = "Learning objectives are to be able to: Recognize when it is appropriate to make a custom function to aid in the reproducibility of an analysis. Use a template to create functions that are reproducible and useable by others. Discuss what aspects of a function make them useable by others. Well documented. Follow a style. Properly engineered", out.width="100%", echo = FALSE}
```

Reply:
Turning the last one into an action verb and adding info about the template. (I adjusted the slide too). We can update if you don't like the phrasing though!

Reply:
for across projects, we could just link out to creating packages (unless you have a different idea for that)